components/DetailsPage/index.js

```diff
@@ -6,37 +6,32 @@ export default function Detail({ comic }) {
   return (
     <main>
       <Link href="/add">zurück zur Übersicht</Link>
-      <div>
-        <>
-          <div key={comic.id}>
-            <div>
-              <Image
-                src={`${comic.thumbnail.path}/portrait_xlarge.${comic.thumbnail.extension}`}
-                alt={comic.title}
-                width={150}
-                height={225}
-                priority
-              />
-
-              <Link href="#">
-                <Image
-                  src="/images/plus.png"
-                  alt="zur Sammlung hinzufügen"
-                  width={100}
-                  height={100}
-                  priority
-                />
-              </Link>
-            </div>
-            <h4>Titel: {comic.title}</h4>
-            <p>
-              <b>EAN/UPC/ISBN:</b> {comic.upc}
-            </p>
-            <p>
-              <b>Beschreibung:</b> {comic.description}
-            </p>
-          </div>
-        </>
+      <div key={comic.id}>
```

Comment:
Because you aren't iterate over a list, you don't need the `key` here and I guess you could then remove at least one `div`.

Reply:
when removing key= than the app doesn't work anymore.

the divs are neccessary because i want the cover and the add button to be displayed next to each other

Reply:
when removing key= than the app doesn't work anymore.

the divs are neccessary because i want the cover and the add button to be displayed next to each other

Reply:
OK i got it now. i will repost it now on slack